ext/js/pages/action-popup-main.js

```diff
@@ -81,6 +81,15 @@ class DisplayController {
 
     // Private
 
+    _updateDisplayModifierKey() {
```

Comment:
Let's target the entire tooltip so we don't end up with "Hold no modifier to scan". In the case of no modifier lets have the string "hover over text to scan".

Reply:
Let's also have this tooltip display "No dictionary enabled" if there's no dictionary enabled